stubstub/global/setup.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-console, global-require */
+// require('ts-node').register({
```

Comment:
this is not required, `ts-jest` already loads `ts-node`

should probably remove it